homeassistant/components/insteon/__init__.py

```diff
@@ -78,12 +76,40 @@ async def async_setup_platforms(hass, config):
             add_on_off_event_device(hass, device)
 
     _LOGGER.debug("Insteon device count: %s", len(devices))
-    register_new_device_callback(hass, config)
+    register_new_device_callback(hass)
     async_register_services(hass)
 
+    conf_data = {**config_entry.data}
```

Comment:
```suggestion
    conf_data = config_entry.data
```

Why make a copy here?